packages/webapp/src/_app/ui/card.tsx

```diff
@@ -8,7 +8,7 @@ interface Props {
 }
 
 export const Card = ({ children, title, titleSize = 1, className }: Props) => (
-    <div className={`bg-white mb-5 p-6 md:p-8 w-full ${className}`}>
+    <div className={`bg-white mb-5 p-6 md:p-8 w-full ${className || ""}`}>
```

Comment:
This was showing "undefined" in the rendered HTML.

Reply:
Yeah, good catch.